src/Distribution/Simple/I18N/GetText.hs

```diff
@@ -208,10 +210,10 @@ getDomainDefine al = findInParametersDefault al "x-gettext-domain-def" "__MESSAG
 getMsgCatalogDefine :: [(String, String)] -> String
 getMsgCatalogDefine al = findInParametersDefault al "x-gettext-msg-cat-def" "__MESSAGE_CATALOG_DIR__"
 
-getPoFilesDefault :: [(String, String)] -> IO [String]
-getPoFilesDefault al = toFileList $ findInParametersDefault al "x-gettext-po-files" ""
+getPoFilesDefault :: Verbosity -> LocalBuildInfo  -> [(String, String)] -> IO [String]
```

Comment:
This change also needs to be guarded by `#if`, because now build with older versions of `Cabal` is broken, e.g.:
```
$ cabal build -w ghc-7.8.4

Building library for hgettext-0.1.40..
[1 of 3] Compiling Internal         ( src/Internal.hs, /Users/abel/bin/src/hgettext/dist-newstyle/build/x86_64-osx/ghc-7.8.4/hgettext-0.1.40/build/Internal.o )

src/Internal.hs:21:18:
    Not in scope: type constructor or class ‘Verbosity’

src/Internal.hs:21:31:
    Not in scope: type constructor or class ‘PackageDescription’
```

Reply:
I pushed a change that imports the missing types unconditionally. Hopefully this should pass the CI.